config/env_sample.hjson

```diff
@@ -252,7 +253,36 @@
                 and membership.id is not null 
                 and REGEXP_EXTRACT(`group`.id, r'.*:(.*)') IN UNNEST(@course_ids)
                 '''
-        }
+        },
+        "canvas_pages": {
+            "query":
+                '''
+                SELECT 'canvas_pages' AS resource_type,
+                CONCAT(CAST(REGEXP_EXTRACT(`group`.id, r'.*:(.*)') AS INT64) - @canvas_data_id_increment, 
+        				'/pages/', 
+        				CAST(JSON_EXTRACT_SCALAR(object.extensions, '$[\'com.instructure.canvas\'][entity_id]') as INT64) - @canvas_data_id_increment) as resource_id,
+                CAST(REGEXP_EXTRACT(membership.id, r'.*:(.*)') AS INT64) AS user_id,
+                cast(null as string) AS user_login_name,
+                CAST(REGEXP_EXTRACT(`group`.id, r'.*:(.*)') AS INT64) AS course_id,
+                JSON_EXTRACT_SCALAR(object.extensions, '$[\'com.instructure.canvas\'][asset_name]') as name,
+                datetime(EVENT_TIME) as access_time
+                FROM event_store.expanded
+                where
+                JSON_EXTRACT_SCALAR(ed_app.json, '$.id') IN UNNEST(['http://m.example.instructure.com/', 'http://example.instructure.com/'])
+                and type = 'NavigationEvent'
+                and object.type = 'Page'
+                and action = 'NavigatedTo'
+                and membership.id is not null
+                and not CONTAINS_SUBSTR(JSON_EXTRACT_SCALAR(extensions_json, '$[\'com.instructure.canvas\'][request_url]'), '/api/v1/')
```

Comment:
It might be nice to have a comment as to why this exclusion was in here. What were these finding?

Reply:
comment added. See [8a39627](https://github.com/tl-its-umich-edu/my-learning-analytics/commit/8a39627b1fbba7a7489335c95de1d4487306ad9e)